wiki/snapshot.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Snapshots
+layout: wiki
+---
+
+# Overview
+Snapshots are a copy of the blockchain at a particular point in time.
+Snapshots can help newer users to get into sync faster than syncing from 0.
+
+# Issues
+* Snapshots are not validated by the wallet.
+* Snapshots are an inherently centralised method of getting into sync.
+* Users may mistakenly install a malicious or corrupted snapshot. 
+If enough users install the bad snapshot then the integrity of the chain could be compromised.
+
+# Using snapshots
+## In the Wallet
+Snapshots can now be downloaded and applied automatically by the wallet.
+
+It is accessible from File -> Snapshot Download or the `-snapshotdownload` [startup option](cmd-options "wikilink")
+
+## Manual method
+1. [Download the snapshot](https://snapshot.gridcoin.us/snapshot.zip)
+2. [Verify the checksum (IMPORTANT)](https://snapshot.gridcoin.us/snapshot.zip.sha256)
+3. Delete the following folders: `txleveldb` & `accrual` from your Gridcoin data directory.
+4. Delete the `blk****.dat` files from your Gridcoin data directory.
+5. Extract the snapshot zip into your Gridcoin data directory.
+6. Restart the wallet and get into sync.
+
+## Creating your own snapshot
+1. Shutdown your wallet.
+2. Create an empty archive in an archive program of your choice.
+3. Add all the `blk****.dat` files to your archive.
+4. Add the `txleveldb` folder to your archive.
+5. Follow the manual method on your other machine.
```

Comment:
Might be good to clarify that you don't use it immediatelyand that you use your own archive instead of the snapshot.gridcoin.us one 
```suggestion
5. When you want to use it, follow the manual method but with your archive```